mne/io/brainvision/brainvision.py

```diff
@@ -54,6 +54,11 @@ class RawBrainVision(_BaseRaw):
         events (stimulus triggers will be unaffected). If None, response
         triggers will be ignored. Default is 0 for backwards compatibility, but
         typically another value or None will be necessary.
+    event_id : dict | None
```

Comment:
I think this is a bit confusingly written ... these values are assigned *in addition* to numeric values that are pulled by default, right?